Python/errors.c

```diff
@@ -1444,12 +1444,14 @@ make_unraisable_hook_args(PyThreadState *tstate, PyObject *exc_type,
 
    It can be called to log the exception of a custom sys.unraisablehook.
 
-   Do nothing if sys.stderr attribute doesn't exist or is set to None. */
+   This assumes file is non-NULL.
```

Comment:
```suggestion
   This assumes 'file' neither NULL nor None.
```

(Just to indicate that it's the argument name)